specs/windows/registry.table

```diff
@@ -0,0 +1,15 @@
+table_name("registry")
+description("All of the Windows registry hives.")
+schema([
+    Column("hive", TEXT, "The parent registry hive where the key and value exist"),
+    Column("key", TEXT, "Name of the key to search for"),
+    Column("subkey", TEXT, "Name of the subkey for a given HIVE and key"),
+    Column("name", TEXT, "Name of the registry value entry"),
+    Column("type", INTEGER, "Type of data associated with the registry value"),
+    Column("data", TEXT, "Data content of registry value"),
+    Column("last_write_time", BIGINT, "timestamp of the most recent registry write"),
```

Comment:
Maybe call this `mtime` to be consistent with a few other columns. I know `mtime` is more POSIX-stat-like diction but it may be helpful in comparing registry modification times and file times.